src/components/news-list/news-list.tsx

```diff
@@ -0,0 +1,39 @@
+import path from 'path';
+import Card from '../Card';
+import style from './news-list.module.scss';
+import { getMarkdowns } from '@/utils/markdown';
+
+export async function NewsList() {
+  // newsの中身を取得
+  const docsDir = path.join(process.cwd(), 'docs', 'news');
+  const docs = await getMarkdowns(docsDir);
+
+  // 一覧を日付でソート
+  docs.sort((a, b) =>
+    a.frontmatter.updatedAt < b.frontmatter.updatedAt ? 1 : -1,
+  );
+
+  // 記事が存在するなら上位3件に絞り、存在しないなら記事がない旨を表示
+  docs.splice(3);
+  if (docs.length === 0) {
+    return <p className={style.newsEmpty}>新着情報はありません</p>;
+  }
```

Comment:
splice は in-place で要素を書き換えちゃうので個人的には使いたくないかも

```suggestion
  if (docs.length === 0) {
    return <p className={style.newsEmpty}>新着情報はありません</p>;
  }
  const topDocs = docs.slice(0, 3);
```